modules/ipfs-cpinner-service/Dockerfile.dataVault

```diff
@@ -0,0 +1,15 @@
+FROM node:10
+
+WORKDIR /app
+
+COPY package.json ./
```

Comment:
This is to have tsc inside the service? Shouldn’t we add tsc as dev inside the module?